spec/src/lib.rs

```diff
@@ -1,8 +1,10 @@
+#![allow(deprecated)]
```

Comment:
what was deprecated?

Reply:
Only `ProgramInstance`.

Reply:
Do you think it is worth to mention it in a comment?

Reply:
Ah, it's just in spec, np

Reply:
Actually no: it's not only in spec. [Example](https://github.com/paritytech/parity-wasm/pull/128/files#diff-b6ba58183c703f8ffcaa54879146b1c2R4)